Minor edits to instructions in readme, requirements, fuzzy and semantic dedupe flags (#548)
NVIDIA/NeMo-Curator: tutorials/dapt-curation/README.md

## Usage

> Please follow the instructions in the README on using docker image and installing the NeMo Curator package. Then, install the following dependencies for running the dapt tutorial:

Reviewer: ```suggestion
Please follow the instructions in the README on using docker image and installing the NeMo Curator package. Then, install the following dependencies for running the DAPT tutorial:
```

> Please follow the instructions in the README on using docker image and installing the NeMo Curator package. Then, install the following dependencies for running the DAPT tutorial:

Reviewer: ```suggestion
Please follow the instructions in NeMo Curator's [README](https://github.com/NVIDIA/NeMo-Curator?tab=readme-ov-file#nemo-framework-container) to run the NeMo Framework Container and install the NeMo Curator package. Then, install the following dependencies for running the DAPT tutorial:
```

```bash
cd /opt/NeMo-Curator/tutorials/dapt-curation/code/
apt update
apt-get install poppler-utils
apt-get install tesseract-ocr

> apt install libtesseract-dev

Reviewer 2: Why were these added , if we added them to pip ? Is pip not enought. 

Asking because `apt update` is a slow command and often folks wont have access to do this. If we can get by using pip , we should do that.

Author: Hey @VibhuJawa , I tried only installing these using pip install , however I ended up getting the following error:
`  File "/usr/local/lib/python3.12/dist-packages/unstructured/partition/pdf.py", line 192, in partition_pdf
    return partition_pdf_or_image(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/unstructured/partition/pdf.py", line 321, in partition_pdf_or_image
    elements = _partition_pdf_or_image_with_ocr(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/unstructured/partition/pdf.py", line 773, in _partition_pdf_or_image_with_ocr
    for page_number, image in enumerate(
  File "/usr/local/lib/python3.12/dist-packages/unstructured/partition/pdf_image/pdf_image_utils.py", line 389, in convert_pdf_to_images
    info = pdf2image.pdfinfo_from_path(filename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/pdf2image/pdf2image.py", line 607, in pdfinfo_from_path
    raise PDFInfoNotInstalledError(`
